include/chatlogpiece.h

```diff
@@ -11,14 +11,14 @@ class chatlogpiece
   chatlogpiece(QString p_name, QString p_showname, QString p_message, bool p_song);
   chatlogpiece(QString p_name, QString p_showname, QString p_message, bool p_song, QDateTime p_datetime);
 
-  QString get_name();
-  QString get_showname();
-  QString get_message();
+  QString get_name() const;
+  QString get_showname() const;
+  QString get_message() const;
   bool get_is_song();
```

Comment:
Not sure why this was left off.

Reply:
I was hesitant to change anything the compiler didn't tell me to, didn't want to silently break something else somewhere.